src/stories/Jobs/CanUpdateJob/endpoint.spec.js

```diff
@@ -0,0 +1,118 @@
+const contextClassRef = requireUtil("contextHelper");
+const randomUser = requireUtil("randomUser");
+const knex = requireKnex();
+const httpServer = requireHttpServer();
+const JobsRepo = requireRepo("jobs");
+
+describe("Test API Jobs/CanUpdateJob", () => {
+  beforeAll(async () => {
+    contextClassRef.user = randomUser();
+    contextClassRef.headers = {
+      Authorization: `Bearer ${contextClassRef.user.token}`, // An authenticated user is making the api call
+    };
+  });
+
+  it("dummy_story_which_will_pass", async () => {
```

Comment:
you can't use such a story name.
Let me know if it's functional. @mahesh-muttinti